api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -515,4 +519,25 @@ public String createBillingProjectName() {
     }
     return projectNamePrefix + randomString;
   }
+
+  @Override
+  public String getCompletedCloneWorkspaceFileTransfer(
+      String workspaceNamespace, String firecloudName) {
+    WorkspacesApi workspacesApi = endUserWorkspacesApiProvider.get();
+    return retryHandler.run(
+        (context) -> {
+          FirecloudWorkspaceDetails fcWorkspaceDetails =
+              workspacesApi
+                  .getWorkspace(
+                      workspaceNamespace,
+                      firecloudName,
+                      FIRECLOUD_WORKSPACE_REQUIRED_FIELDS_FOR_CLONE_FILE_TRANSFER)
+                  .getWorkspace();
+          return fcWorkspaceDetails == null
```

Comment:
when would this be null?

Reply:
If there is no value from Firecloud for FIRECLOUD_WORKSPACE_REQUIRED_FIELDS_FOR_CLONE_FILE_TRANSFER it returns null object.